docker/docker-php-entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -e
+
+composer update --prefer-dist --no-progress --no-suggest --no-interaction
```

Comment:
Usually I don't like this sort of entrypoint because it means every time I'll fire up a new container it'll run composer. I guess you are used to start a new shell in your containers, but in my case usually I do `docker-compose run --rm php70 phpspec run`. In the other hand the set of depencencies won't change often, so I'd prefer to re-add this in the Dockerfile.